### ISSUE TITLE
Emojis are not properly sent

### ISSUE THREAD

**Issue by ghost** [AsamK/signal-cli#352]
My ubuntu terminal certainly is able to display emojis:  ![Screenshot from 2020-09-22 21-20-24](https://user-images.githubusercontent.com/32420579/93927667-7d837b80-fd19-11ea-8633-229aa43c4f6e.png)

However, if I try to send an emoji (as explained in #120) with `signal-cli -u <sender> send -m  💨 <receiver>` (I tried both copy-pasting the code from #120 and also copy-pasting the emoji received from the `receive` command) I receive this message: ![Screenshot from 2020-09-22 21-23-48](https://user-images.githubusercontent.com/32420579/93928035-fda9e100-fd19-11ea-8517-6d56a67f322c.png)

Does anyone have an idea why the emoji is not properly sent? Furthermore, special characters, such as öäüß..., are not properly sent either.

**Comment by AsamK** [comment]
Might be the same as this issue: https://github.com/AsamK/signal-cli/issues/297#issuecomment-613467168

**Comment by p-hofer** [comment]
Thanks for your reply!

My `$LANG` already ended in UTF-8. Anyway, I changed it to `en_US.UTF-8` to double-check that this is not an issue:

```bash
pi@raspberrypi:~ $ echo $LANG
en_GB.UTF-8
pi@raspberrypi:~ $ export LANG="en_US.UTF-8"
```

Nothing changed, I still receive only question marks (see screenshot from first post).

As mentioned in #297 I also tried passing the message through STDIN...
```
echo "Überörtliche Ärztebehörde" | signal-cli -u <sender> send <receiver>
```
... which still gives me the encoding error: 
![Screenshot from 2020-09-24 21-59-03](https://user-images.githubusercontent.com/32420579/94193908-3cbe6a80-feb1-11ea-99d6-76e860e057a9.png)

**Comment by daveriedstra** [comment]
Hi, I'm facing this same issue exactly (which I initially filed in the scli issue mentioned above before realizing it's this). `$LANG` is `en_CA.UTF-8` and signal-cli version is latest (`0.10.4.2` right now). I'm using kitty terminal on pop OS and directly input emojis display fine.

Any thoughts?

Thanks for signal-cli!

**Comment by AsamK** [comment]
Which java version and build are you using?
Also check the encoding java is using: `java -XshowSettings:properties -version 2>&1 | grep encoding`

**Comment by daveriedstra** [comment]
Thanks for looking at this.
```
~ 
❯ apt policy java-common
java-common:
  Installed: 0.72build1
  Candidate: 0.72build1
  Version table:
 *** 0.72build1 500
        500 http://us.archive.ubuntu.com/ubuntu impish/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu impish/main i386 Packages
        100 /var/lib/dpkg/status
~ 
❯ java -XshowSettings:properties -version 2>&1 | grep encoding
    file.encoding = UTF-8
    sun.io.unicode.encoding = UnicodeLittle
    sun.jnu.encoding = UTF-8
~ 
❯ java --version
openjdk 11.0.14.1 2022-02-08
OpenJDK Runtime Environment (build 11.0.14.1+1-Ubuntu-0ubuntu1.21.10)
OpenJDK 64-Bit Server VM (build 11.0.14.1+1-Ubuntu-0ubuntu1.21.10, mixed mode, sharing)
```

**Comment by daveriedstra** [comment]
Seems to be fixed with the recent update to signal-cli v0.10.5, thanks!

```
~ 
❯ signal-cli --version
signal-cli 0.10.5
~ 
❯ apt policy java-common
java-common:
  Installed: 0.72build1
  Candidate: 0.72build1
  Version table:
 *** 0.72build1 500
        500 http://us.archive.ubuntu.com/ubuntu impish/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu impish/main i386 Packages
        100 /var/lib/dpkg/status
~ 
❯ java -XshowSettings:properties -version 2>&1 | grep encoding
    file.encoding = UTF-8
    sun.io.unicode.encoding = UnicodeLittle
    sun.jnu.encoding = UTF-8
~ 
❯ java --version
openjdk 11.0.14.1 2022-02-08
OpenJDK Runtime Environment (build 11.0.14.1+1-Ubuntu-0ubuntu1.21.10)
OpenJDK 64-Bit Server VM (build 11.0.14.1+1-Ubuntu-0ubuntu1.21.10, mixed mode, sharing)
```

![screenshot-2022-04-23_23-04-1650754469](https://user-images.githubusercontent.com/3504098/164948704-ba649f3f-4ec4-4446-a9b1-82ed7161b65e.png)
